python-client/giskard/push/transformation.py

```diff
@@ -0,0 +1,126 @@
+import random
+
+import spacy
+
+# Load the English language model
+nlp = spacy.load('en_core_web_sm')
+
+
+class TextTransformer:
+    def __init__(self):
+        self.nlp = spacy.load("en_core_web_sm")
```

Comment:
What if this fails? This requires the user to have installed the extra spacy module `en_core_web_sm`.